plugin/db/mysql/dump.go

```diff
@@ -45,6 +45,11 @@ const (
 		"-- View structure for `%s`\n" +
 		"--\n" +
 		"%s;\n"
+	tempViewStmtFmt = "" +
```

Comment:
temp does not mean Temporal in general

Reply:
Used word "temporary" instead to be consistent with mysqldump.